pkg/apiserver/controllers/v1/utils.go

```diff
@@ -25,10 +27,24 @@ func getBouncerFromContext(ctx *gin.Context) (*ent.Bouncer, error) {
 	return bouncerInfo, nil
 }
 
+func isUnixSocket(c *gin.Context) bool {
+	if localAddr, ok := c.Request.Context().Value(http.LocalAddrContextKey).(net.Addr); ok {
+		return strings.HasPrefix(localAddr.Network(), "unix")
+	}
+
+	return false
+}
+
 func (c *Controller) AbortRemoteIf(option bool) gin.HandlerFunc {
 	return func(gctx *gin.Context) {
+		if !option {
+			return
```

Comment:
Dont we need to call gctx.Next() before the return?

Reply:
No, calling Next() explicitly is only required if you need to do some processing _after_ the request is served. Otherwise if not stopped, the next middleware is called. And we can remove it from the ApiKey middleware too.